pkg/sysfs/system.go

```diff
@@ -95,6 +155,8 @@ type CPU struct {
 	isolated bool    // whether this CPU is isolated
 }
 
+var _ CPU = &cpu{}
```

Comment:
I think this construct does not produce any benefit here, since both CPU and cpu are part of this same package. Any compile-time error caught by this would be caught anyway, since there must be a function returning a &cpu as a CPU (otherwise having CPU wouldn't make sense with only the single cpu implementing it).

Reply:
Ok, makes sense. Fixed.